src/plugins/controls/public/control_group/component/control_group_component.tsx

```diff
@@ -159,6 +155,17 @@ export const ControlGroup = () => {
                 </DragOverlay>
               </DndContext>
             </EuiFlexItem>
+            {showAddButton && (
+              <EuiFlexItem grow={false}>
+                <EuiButtonIcon
+                  size="s"
+                  iconSize="m"
+                  display="base"
+                  iconType={'plusInCircle'}
+                  onClick={() => controlGroup.openAddDataControlFlyout()}
+                />
```

Comment:
**nit:** We should probably add an `aria-label` here, since it's an icon-only button.

Reply:
Ah, good call! Added!